File: GTTS/INL.UnitLibraryService.Models/SaveUnitAlias_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.UnitLibraryService.Models
{
    public class SaveUnitAlias_Item : ISaveUnitAlias_Item
    {
        public int UnitAliasID { get; set; }
        public long UnitID { get; set; }
        public string UnitAlias { get; set; }
        public short? LanguageID { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public bool IsDefault { get; set; }
        public bool IsActive { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UnitLibraryService.Models/save-unit-alias_item.ts


import { ISaveUnitAlias_Item } from './isave-unit-alias_item';

export class SaveUnitAlias_Item implements ISaveUnitAlias_Item {
  
	public UnitAliasID: number = 0;
	public UnitID: number = 0;
	public UnitAlias: string = "";
	public LanguageID?: number;
	public ModifiedByAppUserID: number = 0;
	public IsDefault: boolean = false;
	public IsActive: boolean = false;
  
}


